passing std::setlocale output to std::locale segfaults,
get the locale with std::locale("") instead.
--
diff --git a/untracked/gtk/gtkmm/application.cc b/untracked/gtk/gtkmm/application.cc
index 0f8d7a0..22c1c8b 100644
--- a/untracked/gtk/gtkmm/application.cc
+++ b/untracked/gtk/gtkmm/application.cc
@@ -74,7 +74,8 @@ static void set_cxx_locale_to_c_locale()
   try
   {
     // Make the C++ locale equal to the C locale.
-    std::locale::global(std::locale(std::setlocale(LC_ALL, nullptr)));
+    // std::locale::global(std::locale(std::setlocale(LC_ALL, nullptr)));
+    std::locale::global(std::locale(""));
   }
   catch (const std::runtime_error& ex)
   {
